### PR TITLE
Handle mysql id decoding in relationships for automations

### PR DESCRIPTION
**Pull request by melohagan**
## Description
ID can be encoded string, so need to decode if needed.
Also added a minor fix when trying to patch a non-existent row. 

Addresses: 
- https://github.com/Budibase/budibase/issues/6048

### CODE REVIEW

**Review by Rory-Powell — APPROVED** [review]
Looks good. Maybe waiting for a review from @mike12345567 would be best before shipping this one

File: packages/server/src/api/controllers/row/index.ts
Comment (mike12345567) [inline packages/server/src/api/controllers/row/index.ts:41]: wouldn't put an exclamation on this - just keep the errors basic.

**Review by mike12345567 — APPROVED** [review]
LGTM - on small comment.

**Pull request merged**
